src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/NoFinalizerCheck.java

```diff
@@ -45,6 +45,31 @@
  * <pre>
  * &lt;module name=&quot;NoFinalizer&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ *  public class Test {
+ *
+ *      protected void finalize() throws Throwable { //Violation
+ *          try {
+ *              System.out.println("overriding finalize()");
+ *          } catch (Throwable t) {
+ *              throw t;
+ *          } finally {
+ *              super.finalize();
+ *          }
+ *      }
+ *
+ *      protected void finalize(Object object) throws Throwable{
+ *          try {
+ *              System.out.println("overriding finalizer() having parameters");
```

Comment:
Remove one whitespace before `System...` to be in line with other statements

Reply:
should i apply this to all the the try and catch and final ?

Reply:
Yes please. Make it consistent through out your code. I still see one case with this line not properly formatted.

Reply:
done